src/testing/jenkins/Fossology_Source_Install.xml

```diff
@@ -128,7 +129,7 @@ cat /etc/issue |grep -iE &apos;ubuntu 12.04&apos;;
 if [ $? -eq 0 ] ; then
   cp ../../composer.json /srv/fossology/test4packageinstall;
   cd /srv/fossology/test4packageinstall;
-  composer install;
+  $mypwd/utils/composer install;
```

Comment:
(unimportant comment:)
Here one should call the makefile?

Is someone actually using this file?